src/poliastro/twobody/angles.py

```diff
@@ -172,7 +172,7 @@ def fp_angle(nu, ecc):
     ecc : float
         Eccentricity.
 
-    Notes
+    Note
```

Comment:
Same as above, I should ask you these changes in the chat. Only made this change because Sphinx automatically detects "Note" and shows it differently.